.github/workflows/android_release.yml

```
@@ -0,0 +1,30 @@
+on:
+  push:
+    branches: [ "main", "android-app" ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: set up JDK 17
+      uses: actions/setup-java@v3
```

actionlint (1.7.10)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)